core/src/main/java/org/apache/druid/common/config/NullHandling.java

```diff
@@ -29,6 +29,7 @@
  * introduced as part of https://github.com/apache/incubator-druid/issues/4349 and the old druid behavior
  * where null values are replaced with default values e.g Null Strings are replaced with empty values.
  */
+@SuppressWarnings("SSBasedInspection") // static field(s) cannot be final because set in an static method
```

Comment:
If the suppression is done for a single field (`INSTANCE`), it should be on the field still.